be/src/storage/lake/load_spill_block_manager.cpp

```diff
@@ -52,15 +71,19 @@ Status LoadSpillBlockManager::init() {
 }
 
 // acquire Block from BlockManager
-StatusOr<spill::BlockPtr> LoadSpillBlockManager::acquire_block(int64_t tablet_id, int64_t txn_id, size_t block_size) {
+StatusOr<spill::BlockPtr> LoadSpillBlockManager::acquire_block(size_t block_size) {
     spill::AcquireBlockOptions opts;
     opts.query_id = _load_id; // load id as query id
     opts.fragment_instance_id =
-            UniqueId(tablet_id, txn_id).to_thrift(); // use tablet id + txn id to generate fragment instance id
+            UniqueId(_tablet_id, _txn_id).to_thrift(); // use tablet id + txn id to generate fragment instance id
     opts.plan_node_id = 0;
     opts.name = "load_spill";
     opts.block_size = block_size;
-    return _block_manager->acquire_block(opts);
+    auto block = _block_manager->acquire_block(opts);
+    if (!block.ok()) {
+        return block.status();
+    }
+    return block.value();
```

Comment:
why not use `return _block_manager->acquire_block(opts);`

Reply:
done